public_html/wp-content/plugins/wordcamp-payments-network/includes/sponsor-invoices-dashboard.php

```diff
@@ -461,3 +464,107 @@ function delete_index_row( $invoice_id ) {
 		)
 	);
 }
+
+/**
+ * Schedule cron to send reminder mails to organizers for unpaid invoices.
+ */
+function schedule_sent_invoice_reminder() {
+	if ( wp_next_scheduled( 'send_invoice_pending_reminder' ) ) {
+		return;
+	}
+
+	wp_schedule_event( time(), 'daily', 'send_invoice_pending_reminder' );
+}
+
+/**
+ * Send reminder to organizer about the unpaid invoice.
+ */
+function send_invoice_pending_reminder() {
+	global $wpdb;
+
+	$table_name    = get_index_table_name();
+	$sent_invoices = $wpdb->get_results( "
+		SELECT blog_id, invoice_id
+		FROM $table_name
+		WHERE status = 'wcbsi_approved'
+		LIMIT 1000
+	" );
+
+	foreach ( $sent_invoices as $invoice_data ) {
+		$blog_id    = $invoice_data->blog_id;
+		$invoice_id = $invoice_data->invoice_id;
+
+		switch_to_blog( $blog_id );
+
+		$invoice         = get_post( $invoice_id );
+		$invoice_sent_at = get_post_meta( $invoice_id, 'Sent at', true );
```

Comment:
This can also be retrieved via `$invoice->{'Sent at'}`, which is a bit more readable. Same for `$last_reminder` and `$invoice_sent_at` below.